### PR TITLE
chore: migrate to owl bot

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/nodejs-appengine-admin/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: .kokoro/continuous/node10/docs.cfg [hunk 1]
```diff
@@ -1,4 +1,4 @@
 env_vars: {
     key: "TRAMPOLINE_BUILD_FILE"
-    value: "github/nodejs-appengine-admin/.kokoro/docs.sh"
+    value: "github/nodejs-appengine/.kokoro/docs.sh"
```
Comment (SurferJeffAtGoogle) [inline .kokoro/continuous/node10/docs.cfg:3]: @bcoe @sofisl Where did the `-admin` originally come from?

Looking at my few changes to `owlbot.py`, I don't see why the output differs.  Do you?

Comment (sofisl) [reply .kokoro/continuous/node10/docs.cfg:3]: @SurferJeffAtGoogle,  I think we needed to merge #29 to fix this issue. Can you try regenerating and see if the issue still appears?

Comment (SurferJeffAtGoogle) [reply .kokoro/continuous/node10/docs.cfg]: I regenerated, and now I only see the -admin suffix being dropped in the system test.

**Comment by bcoe** [comment]
@SurferJeffAtGoogle @sofisl there was a typo in the proto, could you regenerate again?

**Review by JustinBeckwith — APPROVED** [review]

**Pull request merged**